lizrd/train/train_utils.py

```diff
@@ -68,6 +70,16 @@ def get_model(
     ).to(last_gpu)
 
     model = llm.LLM(embedding_layer, encoder_tower, head)
+
+    if load_weights_path:
+        print(f"Loading weights...")
+        assert os.path.exists(
+            load_weights_path
+        ), f"Path {load_weights_path} does not exist"
```

Comment:
We can check it earlier i.e. before model creation

Reply:
right, changed